cubedash/templates/dataset.html

```diff
@@ -4,6 +4,8 @@
 
 {% set extent_geojson = dataset_footprint.__geo_interface__ %}
 
+{% set data_uris = data_uris %}
```

Comment:
Does that do something in jinja?

Reply:
Oh, probably not. I was just copying the patterns from before it!